stable/kube-state-metrics/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 1.6.5
+version: 1.6.6
```

Comment:
These are breaking changes. I would increment the major here. Let's go to `2.0.0` from here-on.